apps/sharedtest/test/unit/telemetry_test.js

```diff
@@ -122,4 +148,41 @@ suite('Telemetry:', function() {
       assert.ok(xhr.onabort);
     });
   });
+
+  suite('getDeviceID', function () {
+
+    setup(function() {
+    });
+
+    teardown(function() {
+    });
```

Comment:
nit: looks like the setup/teardown aren't actually being used, probably better to remove them